CopilotKit/packages/backend/src/lib/openai-assistant-adapter.ts

```diff
@@ -139,19 +153,12 @@ export class OpenAIAssistantAdapter implements CopilotKitServiceAdapter {
     ) {
       run = await this.submitToolOutputs(threadId, forwardedProps.runId, forwardMessages);
     }
-    // submit user message
-    else if (
-      forwardMessages.length > 0 &&
-      forwardMessages[forwardMessages.length - 1].role === "user"
-    ) {
-      run = await this.submitUserMessage(threadId, forwardedProps);
```

Comment:
per the other comment: we want to keep this code here.
The changes should be made inside the `submitUserMessage` function definition